Only in src/rsync-3.2.7: CYGWIN-PATCHES
diff -bur origsrc/rsync-3.2.7/usage.c src/rsync-3.2.7/usage.c
--- origsrc/rsync-3.2.7/usage.c	2022-10-20 20:50:06.000000000 -0400
+++ src/rsync-3.2.7/usage.c	2022-10-25 13:32:39.542022100 -0400
@@ -293,6 +293,7 @@
 		json_line("license", "GPLv3");
 		json_line("caveat", "rsync comes with ABSOLUTELY NO WARRANTY");
 		printf("\n}\n");
+		fflush(stdout);
 		return;
 	}
 
